### PR TITLE
Update `assert_task()`

### PR DESCRIPTION
**Pull request by pat-s**
- allow empty rows and cols (FALSE by default)
- allow checking for a minimum number of rows and cols

cross-ref https://github.com/mlr-org/mlr3featsel/issues/39

### CODE REVIEW

File: R/assertions.R [hunk 1]
```diff
@@ -27,8 +27,18 @@ assert_experiment = function(e, .var.name = vname(e)) {
 #'   Set of allowed feature types.
 #' @param task_properties :: `character()`\cr
 #'   Set of required task properties.
+#' @param allow_empty_rows :: `logical()`\cr
```
Comment (mllg) [inline R/assertions.R]: Is this the same as setting `min_rows` to 1?

File: R/assertions.R [hunk 2]
```diff
@@ -27,8 +27,18 @@ assert_experiment = function(e, .var.name = vname(e)) {
...
+#'   Is a task without observations allowed?
+#' @param minimum_nrows :: `integer()`\cr
+#'   Is a minimum amount of observations required?
+#' @param allow_empty_cols :: `logical()`\cr
```
Comment (mllg) [inline R/assertions.R]: Is this the same as setting `min_cols` to 1?

File: R/assertions.R (hunk 1 above)
Comment (pat-s) [reply R/assertions.R]: Yes, changed it :+1:

**Comment by mllg** [comment]
Thanks!

**Pull request merged**